sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -1362,6 +1379,7 @@ public partial class NetAppVolumeGroupVolume
         public string Name { get { throw null; } set { } }
         public Azure.ResourceManager.NetApp.Models.NetAppNetworkFeature? NetworkFeatures { get { throw null; } set { } }
         public System.Guid? NetworkSiblingSetId { get { throw null; } }
+        public string OriginatingResourceId { get { throw null; } }
```

Comment:
Same here - this reads like a resource identifier from its name. Please change its format to arm-id